config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -436,7 +436,7 @@ periodics:
       resources:
         limits:
           cpu: "6"
-          memory: 16Gi
+          memory: 24Gi
         requests:
           cpu: "6"
           memory: 16Gi
```

Comment:
request.memory needs to be bumped as well

Reply:
Ah missed that one, thanks!